desktop/desktop.scss

```diff
@@ -1,5 +1,6 @@
+@import "variables";
```

Comment:
This is including the `scss/variables.scss` file from this component? I'd rename it to something else, maybe `theme-mixins.scss` (or `theme-variables.scss`, but it looks like it only holds mixing) to avoid a name clash with core's `variables.scss`.